python/social_bot/envs/pr2.py

```diff
@@ -1,20 +1,35 @@
 # Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
 import gym
-import os
 import numpy as np
 import random
-import math
-import PIL
 import gin
 from absl import logging
-
+import time
 from gym import spaces
-import social_bot
-from social_bot import teacher
 from social_bot.envs.gazebo_base import GazeboEnvBase
-from social_bot.teacher import TeacherAction
-import social_bot.pygazebo as gazebo
 import matplotlib.pyplot as plt
+import os
+import social_bot.pygazebo as gazebo
+
+# fix head and left arm and set camera size to 128x128 and format to gray image
+#  these can save some computation
+
+PR2_WORLD_SETTING = [
+    # set camera size and format
+    "//camera//width=256",
+    "//camera//height=256",
+    "//camera//format=L8",
+    # make head fixed
+    "//joint[contains(@name, 'pr2::head_')].type=fixed",
+    # make eye camera sensor focus on the table
+    "//link[@name='pr2::head_tilt_link']/pose=0.033267 -0.003973 1.1676 0.000174 1.0 -0.013584",
+    # using depth camera
+    "//sensor[contains(@name, 'wide_stereo_gazebo_')].type=depth",
```

Comment:
We can take some tricks from FlowNetC or FlowNet2 as the encoding network, might be helpful to extract depth information from the image pair.
https://arxiv.org/abs/1504.06852